NppMenuSearch/NppMenuSearch.csproj.user

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
typically this file user specific file is not committed

Reply:
I took it away from the request.